docs/articles/features/event-pipe-profiler.md

```diff
@@ -0,0 +1,21 @@
+﻿---
+uid: docs.event-pipe-profiler
+name: EventPipeProfiler
+---
+
+# EventPipeProfiler
+
+`EventPipeProfiler` is a cross-platform profiler that allows profile .NET code on every platform - Windows, Linux, macOS. Collected data are exported to trace files (`.speedscope.json` and `.nettrace`) which can be analyzed using [SpeedScope](https://www.speedscope.app/), [PerfView](https://github.com/Microsoft/perfview), and [Windows Performance Analyzer](https://docs.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-analyzer). This new profiler is available from the 0.12.1 version.
```

Comment:
I know that `Visual Studio Profiler` supports it, I am not sure about  `Windows Performance Analyzer`